scripts/test.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import torch
+import torch.nn as nn
+
+cwd = os.getcwd()  # current working directory
+cfp = os.path.dirname(os.path.abspath(__file__))  # current file path
+os.chdir(cfp)
+sys.path.append("..")
+from nn_modules.transformer.top_former import TopFormer
+os.chdir(cwd)
+
+top_former = TopFormer(num_layers=6, d_model=512, nhead=8, num_classes=3)
+
+# TODO: remove this part
+for param in top_former.parameters():
+    if param.dim() > 1:
+        nn.init.xavier_uniform_(param)
+
+model_file_path = 'top_eleven.pt'
+
+# TODO: remove this line
+torch.save(top_former.state_dict(), model_file_path)
```

Comment:
More file check to avoid interrupted by crash.

Reply:
Done.

A helper function `check_model_file_path()` is added to `utils`.